timestamp-implementation-advice.xml

```diff
@@ -85,87 +85,129 @@
         <t>In this document we describe the properties of various available time values on modern operating systems, discuss the trade-offs of using one over the other, and provide guidance to help implementors make an informed choice with some real-life examples.</t>
      </section>
 
-      <section title="Keeping Time: Different Clocks">
-          <t> Because time is relative to an observer, there cannot be a
-              universally agreed upon time. At best we can achieve an
-              approximation by updating our own observed time with a
-              common reference time shared with other observers.</t>
-          <t> As this reference time is what we naively assume clocks on a
-              wall are showing, we shall call it the "wall time." For most
-              applications, it is based on the Universal Coordinated Time
-              (UTC), an international standard time determined by averaging
-              the output of several high-precision time-keeping devices.
-              However, as UTC is following Earth's solar time, it occasionally
-              needs to be adjusted through leap seconds.</t>
-          <t> An individual computer system's preception of time differs from
-              this idealized wall time. Staying close to it requires some
-              effort that comes with its own set of drawbacks. Systems
-              therefore provide access to different types of clocks with
-              different properties. Unfortunately, there is no standard
-              terminology and definitions for these types. For the purpose of
-              this document, we therefore define three different kinds of
-              clocks that a system may or may not provide.</t>
+    <section title="Keeping Time: Wall Time and Various Clocks">
+      <t>
+        Even if one ignores the intricacies introduced by the relative nature
+        for a Newtonian view of an absolute time ticking away at a constant
+        rate, the varying inaccuracies of all practical time-keeping devices
+        make agreeing on a precise time difficult.
```

Comment:
Very pittoresque and perhaps even a bit windy :)
One could also just say: "The inaccuracies of time-keeping devices  make agreeing on a precise time difficult"